frontend/public/favicon/site.webmanifest

```diff
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```

Comment:
Could we move this file to the root of `public/` and name it `manifest.json`? In my experience that's a common location and the file extension will help with formatting automatically.

Reply:
I can definitely move it but i'll have to see if renaming it will have any side effects since like favicon.ico some browsers might automatically look for a certain name. I'll let you know when i figure that out.